docs/reference/configuration/uds-operator/overview.md

```diff
@@ -4,7 +4,7 @@ sidebar:
     order: 1
 ---
 
-![UDS Operator Overview Flows](https://github.com/defenseunicorns/uds-core/blob/main/docs/.images/diagrams/uds-core-operator-custom-resources.svg?raw=true)
+![UDS Operator Overview Flows](https://github.com/defenseunicorns/uds-core/blob/main/docs/.images/diagrams/uds-core-operator-overview.svg?raw=true)
```

Comment:
To validate this link, replace the branch name: https://github.com/defenseunicorns/uds-core/blob/diagram-update/docs/.images/diagrams/uds-core-operator-overview.svg?raw=true

Note that for whatever reason the raw link like this doesn't show the fonts but you can see in the GitHub PR file review that they are present, or if opening locally.